### ISSUE TITLE
unable to download a latest-v.x version

### ISSUE THREAD

**Issue by xenoterracide** [node-gradle/gradle-node-plugin#184]
https://nodejs.org/dist/

```kts
node {
  download.set(true)
  version.set("latest-v14.x")
}
```

**Comment by deepy** [comment]
This is the same issue as #54

There's a workaround in https://github.com/node-gradle/gradle-node-plugin/issues/54#issuecomment-581950868

But the short of it is that you'll have to use `distBaseUrl = null` and manually add an `IvyRepository`, if you want dynamic versions e.g. `14.+` you'll also need the `ComponentMetadataVersionLister` from there (or a similar one)

But in general I advise strongly against dynamic versions, reproducible builds are good and if you want to run bleeding edge versions something like [ben-manes/gradle-versions-plugin](https://github.com/ben-manes/gradle-versions-plugin) + [patrikerdes/gradle-use-latest-versions-plugin](https://github.com/patrikerdes/gradle-use-latest-versions-plugin) and generating PR/MRs.
Which unfortunately also requires you to use `distBaseUrl = null` and manually add an `IvyRepository` with a `ComponentMetadataVersionLister` but at least you'll have reproducible builds and easy one-click upgrades :)

**Comment by xenoterracide** [comment]
well, I don't want to run bleeding edge builds, this is tangentially related to #183 as I really just want to say `NodeVersion.of(14)` since that would be the latest stable.

**Comment by deepy** [comment]
Oh I missed the follow-up, but yeah, https://github.com/node-gradle/gradle-node-plugin/issues/54#issuecomment-581950868 sounds like what you want, `NodeVersion.of(14)` is not strictly equivalent to `14.+` but you can fulfill `NodeVersion.of(14)` with `version.set("14.0.0")` since even the earliest version of 14 satisfies `of(14)`, whereas `14+` would give you the latest version of 14.

**Comment by xenoterracide** [comment]
yeah, but the equivalent in java will grab the latest unless you already have a copy on the path that fulfills it.
